src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js

```diff
@@ -6,10 +6,13 @@ Cypress.Commands.add('checkSearch', () => {
   cy.axeCheck();
 
   // Search
+  cy.get('#street-city-state-zip', { timeout: 10000 })
+    .should('exist')
+    .clear();
   [...city].forEach(char => {
     cy.get('#street-city-state-zip')
       .should('not.be.disabled')
-      .type(char);
+      .type(char, { waitForAnimations: true, force: true });
```

Comment:
According to the Cypress docs, `force: true` disables waiting for actionability, so I'm not sure why we need `waitForAnimations: true` here. I guess it doesn't hurt anything though.

I know this was done in an earlier PR, but I'd also like to understand why we needed to loop on every character in the string. Could you please explain, ideally in a comment in the code? Thanks!

Reply:
I think we can take out the `force: true` here. I was testing a few different fixes, and I don't think this is needed anymore.

Looping to enter each character is a workaround to an [apparent typing bug in Cypress](https://github.com/cypress-io/cypress/issues/5480) that should be fixed when we upgrade to Cypress 6.1, which we'll be doing in the next few weeks.

Reply:
Thanks for the info about the typing bug. That should probably be posted to the frontend team. Please add a comment to document why this is being done, and a reminder to remove it after the upgrade.